cmd/build/helmify/static/README.md

```diff
@@ -97,6 +97,7 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 | controllerManager.metricsPort                | Metrics port for controller manager                                                    | `8888`                                                                    |
 | controllerManager.priorityClassName          | Priority class name for controller manager                                             | `system-cluster-critical`                                                 |
 | controllerManager.exemptNamespaces           | The namespaces to exempt                                                               | `[]`                                                                      |
+| controllerManager.exemptNamespacePrefixes    | The namespace prefixes to exempt                                                       | `[]`                                                                      |
```

Comment:
nit: s/The namespace prefixes to exempt/The namespace prefixes to exempt by the admission webhook

Reply:
Would it make sense to also change `exemptNamespaces` to something along the lines of
> The exact namespaces to exempt by the admission webhook

Reply:
yes! Can you pls update it as well?

Reply:
Done 🎉